bindgen-cli/options.rs

```diff
@@ -111,12 +127,11 @@ struct BindgenCommand {
     /// Mark any enum whose name matches REGEX as a global newtype.
     #[arg(long, value_name = "REGEX")]
     newtype_global_enum: Vec<String>,
-    /// Mark any enum whose name matches REGEX as a Rust enum.
-    #[arg(long, value_name = "REGEX")]
-    rustified_enum: Vec<String>,
-    /// Mark any enum whose name matches REGEX as a non-exhaustive Rust enum.
-    #[arg(long, value_name = "REGEX")]
-    rustified_non_exhaustive_enum: Vec<String>,
+    /// Mark any enum whose name matches the provided regex as a Rust enum. This parameter takes
+    /// options in the shape REGEX or REGEX=OPTIONS where OPTIONS can be a comma separated list of
```

Comment:
So this disallows a regex without options but with a `=` character, right? I guess we don't need that, but just noting it.

Reply:
Yes, that's correct. If you provide no equal sign, it uses the empty option set.